common/Firewall/sec_flash.h

```diff
@@ -18,6 +18,9 @@
 #ifndef SECURETASKS_PROTECTED_CODE_H_
 #define SECURETASKS_PROTECTED_CODE_H_
 
+#include "flash_config.h"
+#include "wallet.h"
+
```

Comment:
Since we haven't made any changes to this file, can you mention the intent of adding these includes?

Reply:
These includes are required in this header file, otherwise including sec_flash.h by itself results in build failures.

Refer : https://github.com/Cypherock/x1_wallet_firmware/actions/runs/5308445320/jobs/9607939780